about.html

```diff
@@ -0,0 +1,148 @@
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link">News</a>
+            </li>
+          </ul>
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </header>
+      <article class="intro">
+        <h1 class="creativetxt2">
+          CONSTRUCTION ENGINEERING <br />
+          GLOBAL SUMMIT <br />
+          2020
+        </h1>
+        <p class="joyfultxt2">
+          The Global Summit brings together the community of exparts, academics,
+          and activits who comprise the creative Commons affiliate network in a
+          different country every two. This year, we hope to expand our
+          invitation list including organozations and individuals.so if you are
+          active and engaged in world of construction we would like to hear from
+          you.
+        </p>
+        <p class="abouttxt">
+          please contact us via email for any further questions about CES GLOBAL
+          SUMMIT
+        </p>
+        <p class="text-center mail">
+          <a href="mailto:ntihinyukae@gmail.com">ntihinyukae@gmail.com</a>
+        </p>
+      </article>
+    </section>
+    <article class="card p-5 my-5 mx-4">
+      <h2 class="logo-text text-center">
+        Creative Construction Engineering <br />
+        Summit 2020 Logo
+      </h2>
+      <p class="logo-decide text-center">
+        The logo of Construction Engineering summit was selected among one's
+        that was in logo competition from 2nd to 6th june.
+      </p>
+
+      <div class="card cardsumittxt p-3 logo-wrapper">
+        <img
+          class="logoimg"
+          src="img/summitlogo.jpeg"
+          alt="logo"
+          width="100px"
+        />
+        <p cardsumit>
+          CES <br />
+          Construction Engineering <br />
+          summit 2020
+        </p>
+      </div>
+    </article>
+
+    <article class="card graduates p-lg-3 pb-lg-5 border-1 m-0">
+      <p class="graduates-header h2 text-center mt-4">
+        See the past CES Construction Engineering summit 2020
+      </p>
+      <h2 class="baseline text-center text-white fw-bold"></h2>
+      <p class="text-center mt-4">
+        take a look at the last two CC Global Summits which took place <br />
+        in Fiji and kigali Rwanda
+      </p>
+
+      <div
+        class="d-lg-flex my-5 px-5 justify-content-center justify-content-lg-center"
+      >
+        <div class="gradPhoto mt-4 me-lg-2 text-center align-items-center">
+          <p class="h1 year">2010</p>
+          <p class="below-year">CE Global Summit in amsterdam Netherlands</p>
+        </div>
+
+        <div class="gradPhotoes mt-4 ms-lg-2 text-center align-items-center">
+          <p class="h1 year">2015</p>
+          <p class="below-year">CE Global Summit in Kigali, Rwanda</p>
+        </div>
```

Comment:
- [ ] Please change the color of the text on these images to match the given template.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/227250231-c1685c29-7d3b-4a03-96a2-8e9efa85c77a.png) | ![image](https://user-images.githubusercontent.com/62290545/227250313-bf9228d3-9609-4bfb-a822-b5d4407347a0.png) |